matplotcheck/tests/test_base.py

```diff
@@ -1,53 +1,113 @@
 """Tests for the base module"""
 import pytest
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotcheck.base import PlotTester
 
 
-def test_line_plot(pt_line_plt):
+@pytest.fixture
+def pd_df():
```

Comment:
doesn't this below on the conftest.py file? it seems like it may be redundant to have here too.